CHANGES.md

```diff
@@ -19,6 +46,14 @@
     pub fn add_rule(&mut self, (name, name_span): (String, Span), actiont: Option<String>) {
   ```
 
+* `YaccParserError` has been renamed to `YaccGrammarError` and changed from an
+  `enum` (which frequently changed, breaking semver) to a `struct`. The `kind`
+  field is now internal (so won't break semver as often).
+
+* `LexBuildresult` returns on failure `Err(Vec<LexBuildError>)` rather than
+  `Err(LexBuildError)`.
+
```

Comment:
This one I still need to look at and see if we need to give `LexBuildResult` the same treatment regarding `spans` field and `kind` as we did to `YaccGrammarError`, so there may be one more semver change here perhaps.

Reply:
Ah, OK. Should I delete this line or ... ?

Reply:
No I think what is written shouldn't be changing, 
Just wanted to point out one last semver change might be needed.

Reply:
Got it!